src/PresentationalComponents/RulesTable/RulesTable.js

```diff
@@ -92,6 +92,7 @@ const RulesTable = ({ isTabActive }) => {
     });
   workloads &&
     (options = { ...options, ...workloadQueryBuilder(workloads, SID) });
+  pathway && (options = { ...options, pathway });
```

Comment:
I'm not sure about the opinion of other FE engineers, but I find using short-circuiting inside LHS of assignments very hard to read. Don't get me wrong, I like concise code, but not at the cost of readability. I'd personally prefer if this assignment was either wrapped in `if` or the `&&` was inside RHS of the assignment. What do you think?

Reply:
I don't mind changing it, but I was just keeping with the style of the surrounding code.

Reply:
Oki, this might be something for a separate PR.

Reply:
If I change this line, then the lines above should be changed as well I guess?  To make it consistent?  But this assignment style for options has been used throughout the project, not just in this file.  If we change it here then should we change it in the other files too?  I vote we keep it consistent with existing code.  Its readable enough.

Reply:
I agree with you both. :D 

We should keep established patterns, where they exist, but the assignments here are hard to read done this way. A better and more common through out all apps would be something like:

```js
const options = {
   ...selectedTags?.length && { tags: selectedTags.join(',') } : {},
   ...workloads && workloadQueryBuilder(workloads, SID) : {},
   ...pathway && { pathway } : {}
};
```

<sup>*</sup> This is an example, please do double check before applying.

Reply:
@bastilian yes, this format would be perfect, I think you have redundant `: {}` at the end of each line

Reply:
I'll leave it as is, and go through the project at another time and change all the option assignments.  It could probably be put into some sort of library function.

However, I was curious about the failing checks.  Should this be addressed before merging the PR?  I'm not sure why the linter tests are failing though, it seems be something outside of this PR that's causing it to fail.  Your thoughts @leSamo @bastilian ?

Reply:
Yea, the deps are failing, I cannot reproduce the error locally :thinking: